eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -53,6 +53,17 @@ jobs:
           node common/scripts/install-run-rush.js install
         displayName: "Install dependencies"
 
+      - script: |
+          node index.js --buildId "$(Build.BuildNumber)"
+        displayName: "Update package versions"
+        workingDirectory: $(System.DefaultWorkingDirectory)/eng/tools/generate-dev-builds
+        condition: eq(variables['DevBuildRun'],'true')
```

Comment:
Were do you set DevBuildRun to true?

Reply:
I set this in the pipeline we run - like i tried running a build from js - client pipeline and while pointing it to this PR i set the DevBuildRun queue time variable to true.